speedy-recovery-client/src/components/secured/conversation/ConversationPage.js

```diff
@@ -94,20 +95,22 @@ class ConversationPage extends Component {
   }
 
   componentDidMount() {
+    this._isMounted = true;
     this.setMessageList();
     //Check for new messages every 3 seconds
     this.timer = setInterval(() => {
       this.setMessageList();
-    }, 3000);
+    }, 5000);
   }
 
   componentWillUnmount() {
+    this._isMounted = false;
     this.timer && clearTimeout(this.timer);
   }
 
   //Populate the history of messages in this conversation on the page
   setMessageList = () => {
-    if (this.props.location) {
+    if (this.props.location && this._isMounted) {
       this.setState({ title: this.props.location.state.title });
```

Comment:
What is this doing?

Reply:
The isMounted is removed and the timer is changed to 5 seconds instead.